packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -507,6 +507,37 @@ export class StripeHelper {
     return this.stripe.invoices.pay(invoice.id, { paid_out_of_band: true });
   }
 
+  /**
+   * Update the customer object to add customer's PayPal billing address.
+   * 
+   * @param customer_id 
+   * @param city 
+   * @param country 
+   * @param line1 
+   * @param line2 
+   * @param postal_code 
+   * @param state 
+   */
+  async updateCustomerBillingAddress(
+    customer_id: string,
```

Comment:
This is a lot of arguments, it would be much cleaner to pass this in as an object.